service/create/cert_signer.go

```diff
@@ -0,0 +1,73 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type issueResponse struct {
+	Certificate  string
+	PrivateKey   string
+	IssuingCA    string
+	SerialNumber string
+}
+
+// Issue generates a certificate using the PKI backend signed by the certificate
+// authority associated with the configured cluster ID.
+func (s *Service) Issue(config CertificateSpec) (issueResponse, error) {
```

Comment:
This looks like the code in the certctl libs. I thought we can simply reuse this. There is no reason to copy/paste all of the code.